### PR TITLE
5725 initial CI documentation

### PR DESCRIPTION
**Pull request by donsizemore**
## New Contributors

Welcome! New contributors should at least glance at [CONTRIBUTING.md](/CONTRIBUTING.md), especially the section on pull requests where we encourage you to reach out to other developers before you start coding. Also, please note that we measure code coverage and prefer you write unit tests. Pull requests can still be reviewed without tests or completion of the checklist outlined below. Thanks!

## Related Issues

- connects to #5725

## Pull Request Checklist

- [ ] Unit [tests][] completed
- [ ] Integration [tests][]: None
- [ ] Deployment requirements, [SQL updates][], [Solr updates][], etc.: None
- [ ] [Documentation][docs] completed
- [ ] Merged latest from "develop" [branch][] and resolved conflicts

[tests]: http://guides.dataverse.org/en/latest/developers/testing.html
[SQL updates]: http://guides.dataverse.org/en/latest/developers/sql-upgrade-scripts.html
[Solr updates]: https://github.com/IQSS/dataverse/blob/develop/conf/solr/7.3.0/schema.xml
[docs]: http://guides.dataverse.org/en/latest/developers/documentation.html
[branch]: http://guides.dataverse.org/en/latest/developers/branching-strategy.html

### CODE REVIEW

File: doc/sphinx-guides/source/developers/testing.rst [hunk 1]
```diff
@@ -224,6 +224,17 @@ One way of generating load is by downloading many files. You can download :downl
...
+Continuous Integration
+~~~~~~~~~~~~~~~~~~~~~~
+
+The Dataverse Project currently makes use of two Continuous Integration platforms, Travis and Jenkins.
+
+Travis builds are configured via :download:`.travis.yml <../../../../.travis.yml>` and a `GitHub webhook <https://docs.travis-ci.com/user/notifications/#configuring-webhook-notifications>`; build output is viewable at https://travis-ci.org/IQSS/dataverse/builds
+
+Our Jenkins config is a work in progress and may be viewed at https://github.com/IQSS/dataverse-jenkins/ A corresponding GitHub webhook is required. Build output is viewable at https://jenkins.dataverse.org/
+
+As always, pull requests to improve our continuous integration configurations are welcome.
```
Comment (pdurbin) [inline doc/sphinx-guides/source/developers/testing.rst]: @donsizemore I'm glad you're characterizing the dataverse-jenkins (the git repo) and https://jenkins.dataverse.org as a work in progress. I think you've stubbed this out very well. Thanks!!

**Comment by pdurbin** [comment]
> except for the connection through PRs

@qqmyers thanks, can you please expand on this a bit? Right now we have Travis integrated such that we see when checks done by Travis pass or fail, like this example from pull request #5621:

Screen Shot 2019-04-11 at 10 46 17 AM

Are you saying that from GitHub you see "All checks have passed" (or failures) based on testing done on Jenkins?

**Comment by qqmyers** [comment]
@pdurbin - I just meant that I use Dataverse's build system when I submit PRs. Otherwise, when I'm developing, I'm relying on QDR's build system and checking test results in our jenkins console, etc. W.r.t. integration, I'm not sure how its done in QDR, but new builds are triggered by my commits to github - I don't think the results get back to github (not sure I've looked).

**Comment by 4tikhonov** [comment]
Hi @pdurbin and others, we're not going to create a clone of Dataverse, the idea that all our contribution should go to the master branch. And we already have first priorities for development in DataverseEU.

**Comment by pdurbin** [comment]
@4tikhonov yes, of course, and we're looking forward to pull requests from you and the rest of the DataverseEU developers! What I was trying to say at https://github.com/IQSS/dataverse/issues/5725#issuecomment-482083170 is that I'm wondering if I can get an account on your CI pipeline system. I showed your slides to @djbrooke this morning and said, "It's like Slava's throwing a continuous integration party and I hope I get invited!" 🎉 🎈 Is it possible for me to trigger a build on your system and see if I broke anything when I push code? Do I need an account? Can I get an account? 😄 I'd love to play in your playground, if you're ok with that. 😄

**Comment by 4tikhonov** [comment]
@pdurbin, sure, I think it's not a problem. We're discussing now to setup two different DataverseEU instances, first on CESSDA Google Cloud (for sure) and second on Kubernetes cluster in Poland (it has to be finally decided). Guys from Poland also interested to deliver polish translation, so just stay tuned. :)

**Pull request merged**